packages/backend-core/src/events/publishers/backup.ts

```diff
@@ -10,3 +15,12 @@ export async function appBackupRestored(backup: AppBackup) {
 
   await publishEvent(Event.APP_BACKUP_RESTORED, properties)
 }
+
+export async function appBackupTriggered(appId: string, name?: string) {
+  const properties: AppBackupTriggeredEvent = {
+    appId: appId,
+    backupName: name,
```

Comment:
Can we add a trigger field as well? So we can distinguish between manual, on publish or on schedule in future

Reply:
Makes sense, worth knowing!